2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Configure stats pid to 12260
2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Loading settings from C:\Users\glebk\.config\wandb\settings
2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Loading settings from D:\Study\BP\wandb\settings
2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'sweeps-hyperparameter-tuning-in-privacy-preserving-machine-learning', 'entity': 'dpnerds', 'sweep_id': 'hvu7c44i', 'root_dir': 'D:\\Study\\BP', 'run_id': '9jkt540z', 'sweep_param_path': 'D:\\Study\\BP\\wandb\\sweep-hvu7c44i\\config-9jkt540z.yaml'}
2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-02 00:56:29,965 INFO    MainThread:12260 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'scripts\\sweep_training.py', 'program': 'D:\\Study\\BP\\scripts\\sweep_training.py'}
2024-03-02 00:56:29,966 INFO    MainThread:12260 [wandb_init.py:_log_setup():506] Logging user logs to D:\Study\BP\wandb\run-20240302_005629-9jkt540z\logs\debug.log
2024-03-02 00:56:29,966 INFO    MainThread:12260 [wandb_init.py:_log_setup():507] Logging internal logs to D:\Study\BP\wandb\run-20240302_005629-9jkt540z\logs\debug-internal.log
2024-03-02 00:56:29,966 INFO    MainThread:12260 [wandb_init.py:init():546] calling init triggers
2024-03-02 00:56:29,966 INFO    MainThread:12260 [wandb_init.py:init():552] wandb.init called with sweep_config: {'batch_size': 30, 'epochs': 5, 'l2_norm_clip': 0.15, 'learning_rate': 9.455e-05, 'microbatches': 1, 'noise_multiplier': 5}
config: {'learning_rate': 1e-05, 'l2_norm_clip': 0.1, 'batch_size': 15, 'noise_multiplier': 0.1, 'microbatches': 1, 'dataset': 'heart_2022_no_nans', 'epochs': 5}
2024-03-02 00:56:29,966 INFO    MainThread:12260 [wandb_init.py:init():597] wandb.init() called when a run is still active
2024-03-02 00:56:29,976 ERROR   MainThread:12260 [wandb_init.py:init():1171] error
Traceback (most recent call last):
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\wandb_init.py", line 1144, in init
    run = wi.init()
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\wandb_init.py", line 599, in init
    tel.feature.init_return_run = True
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\lib\telemetry.py", line 42, in __exit__
    self._run._telemetry_callback(self._obj)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 691, in _telemetry_callback
    self._telemetry_flush()
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 702, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 101, in _publish_telemetry
    self._publish(rec)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\lib\sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\lib\sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\lib\sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "D:\Study\BP\venv\lib\site-packages\wandb\sdk\lib\sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
